samples/sensor/apds9960/boards/nucleo_f401re.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2021 Guðni Már Gilbert
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * i2c interface for the Nucleo F401RE and APDS9960
+ */
+
+ &i2c1 {
```

Comment:
I wonder if there would be ways to set this much more generic using arduino_i2c port reference, and avoid introducing board specific files.
Could you have a check to `samples/sensor/amg88xx/app.overlay` and see if same approach could be used here ?

Reply:
That would be nice. I will take a look.

Reply:
app.overlay is not right way here because we have a board in the tree equipped with this sensor. It is better to add a generic shield support for APDS9960 breakout-boards.

Reply:
Something like shield support for [SSD1306 displays](https://github.com/zephyrproject-rtos/zephyr/tree/master/boards/shields/ssd1306)